examples/http/hpe_msa_san_hardware_sensors_voltage_current.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Domotz Custom Driver 
+ * Name: HPE MSA SAN HW Sensors Voltage and Current 
+ * Description: This script retrieves voltage and current sensor data from an HPE MSA SAN device
+ * 
+ * Communication protocol is HTTPS 
+ * 
+ * Tested with HPE MSA 2050 SAN version: VL270P005
+ * 
+ * Creates a Custom Driver Table with the following columns:
+ *     - Value: The measured voltage or current value in volts
+ *     - Status: The status of the sensor
+ * 
+ */
+
+// Variable to store the session key
+var sessionKey;
+
+// Create a Custom Driver table to store to store voltage and current sensor data
+var table = D.createTable(
+    "Voltage and Current Sensors",
+    [
+        { label: "Value", unit: "V", valueType: D.valueType.NUMBER},
```

Comment:
The Value unit is not correct for Current. I suggest an extra column for current if grouping is possible